idl/cherami_server/metadata.thrift

```diff
@@ -614,6 +648,19 @@ service MetadataService extends MetadataExposable {
       2: IllegalStateError illegalStateError
       3: shared.InternalServiceError internalServiceError
     )
+
+  ListDestinationExtentsResult listDestinationExtents(1: ListDestinationExtentsRequest request)
+    throws (
+      1: shared.BadRequestError requestError
+      2: shared.InternalServiceError internalServiceError
+    )
+
+  ReadConsumerGroupExtentsLiteResult readConsumerGroupExtentsLite(1: ReadConsumerGroupExtentsRequest request)
```

Comment:
Let's use the different request struct for lite API.  Reusing the request struct between 2 different APIs might get confusing later.